src/vs/workbench/api/common/extHostQuickOpen.ts

```diff
@@ -67,11 +67,12 @@ export function createExtHostQuickOpen(mainContext: IMainContext, workspace: IEx
 			const instance = ++this._instances;
 
 			const quickPickWidget = proxy.$show(instance, {
-				placeHolder: options && options.placeHolder,
-				matchOnDescription: options && options.matchOnDescription,
-				matchOnDetail: options && options.matchOnDetail,
-				ignoreFocusLost: options && options.ignoreFocusOut,
-				canPickMany: options && options.canPickMany
+				title: options?.title,
+				placeHolder: options?.placeHolder,
+				matchOnDescription: options?.matchOnDescription,
+				matchOnDetail: options?.matchOnDetail,
+				ignoreFocusLost: options?.ignoreFocusOut,
+				canPickMany: options?.canPickMany,
 			}, token);
```

Comment:
this might be able to be
```suggestion
				...options
			}, token);
```

Reply:
There could be other properties on `options` that we don't support and are only used by the extension. `...` would copy these too. 🤷‍♂️

Reply:
Probably not so important on the options object.